### PR TITLE
Resolve some PMD issues and error on newly introduced errors

### PR DESCRIPTION
**Pull request by grimreaper**

### CODE REVIEW

File: code/src/java/plugin/overland/model/RoomBoardFactory.java [hunk 1]
```diff
@@ -69,10 +72,12 @@ public static RoomBoard load(File dataDir)
 
 					methodSet = null;
 					dt = null;
-				}
-				catch (Exception e)
+				} catch (Exception e)
```
Comment (thpr) [inline code/src/java/plugin/overland/model/RoomBoardFactory.java]: Curly Brace to own file

File: code/src/java/plugin/overland/model/TravelMethodImplementation.java [hunk 2]
```diff
@@ -852,7 +897,10 @@ public int getSize()
 		public Object getElementAt(int index)
 		{
 			if (selectedMethod == null || index < 0 || getList(selectedMethod).size() <= index)
+
```
Comment (thpr) [inline code/src/java/plugin/overland/model/TravelMethodImplementation.java]: Why the blank line?

File: code/src/java/plugin/overland/model/TravelMethodImplementation.java [hunk 3]
```diff
@@ -803,10 +840,16 @@ private void fireMethodChanged(MethodModel source, Method previousMethod)
 				end = Math.min(end, previousSize);
 				// increment start for each identical element at the start of the paces lists
 				for (int i = 0; i < end && previousList.get(i).equals(selectedList.get(i)); i++, start++)
+
+				{
 					;
+				}
 				// decrement end for each identical element at the end of the paces lists
 				for (int i = end - 1; i > start && previousList.get(i).equals(selectedList.get(i)); i--, end--)
+
```
Comment (thpr) [inline code/src/java/plugin/overland/model/TravelMethodImplementation.java]: Why the blank line?

File: code/src/java/plugin/overland/model/TravelMethodImplementation.java [hunk 4]
```diff
@@ -803,10 +840,16 @@ private void fireMethodChanged(MethodModel source, Method previousMethod)
 				end = Math.min(end, previousSize);
 				// increment start for each identical element at the start of the paces lists
 				for (int i = 0; i < end && previousList.get(i).equals(selectedList.get(i)); i++, start++)
+
+				{
```
Comment (thpr) [inline code/src/java/plugin/overland/model/TravelMethodImplementation.java]: I'm lonely.  Can I please be next to my for statement?

File: code/src/java/translation/util/Tips.java [hunk 5]
```diff
@@ -312,15 +326,16 @@ private static void generateTips(File rootDirectory,
 		finally
 		{
 			if (translationReader != null)
+			{
 				try
 				{
 					translationReader.close();
-				}
-				catch (IOException e)
+				} catch (IOException e)
```
Comment (thpr) [inline code/src/java/translation/util/Tips.java]: brace!

File: data/pathfinder/paizo/roleplaying_game/occult_adventures/.!24009!oa_abilities_race.lst [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# CVS $Revision: $ $Author: $ -- Thu Nov 26 22:25:37 2015 -- reformated by PCGen PrettyLST v6.05.01
```
Comment (thpr) [inline data/pathfinder/paizo/roleplaying_game/occult_adventures/.!24009!oa_abilities_race.lst]: Woah!  Data file is s no-no, especially a temporary one

File: code/src/java/pcgen/system/ConfigurationSettings.java [hunk 7]
```diff
@@ -93,19 +85,6 @@ public static void setCountry(String country)
...
 
-	/**
-	 * @return the current user directory
-	 */
-	public static String getUserDir()
-	{
-		return SystemUtils.USER_DIR;
-	}
-
-	public static ConfigurationSettings getInstance()
```
Comment (thpr) [inline code/src/java/pcgen/system/ConfigurationSettings.java]: I don't see the value of eliminating this and forcing many more nulls into other places in the code.  I think localizing the use of null is better

**Comment by grimreaper** [comment]
I'm going to split this up into separate PRs.

**Pull request closed without merging**